com.cocoatech.deckr.yaml

```diff
@@ -0,0 +1,129 @@
+app-id: com.cocoatech.deckr
+runtime: org.gnome.Platform
+runtime-version: '43'
+sdk: org.gnome.Sdk
+command: deckr
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --filesystem=home
+  - --filesystem=xdg-desktop
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-download
+  - --filesystem=xdg-music
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-videos
+
+modules:
+  - name: deckr
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    build-commands:
+      # - mkdir -p Deckr
+      # - tar -xf Deckr-Linux-Portable.tar.gz -C Deckr
+      - cp -r Deckr /app/Deckr
+      - echo $(ls /app/Deckr)
+      - chmod +x /app/Deckr/deckr
+      - mkdir -p /app/bin
+      - ln -s /app/Deckr/deckr /app/bin/deckr
+      - mkdir -p /app/share/metainfo
+      - cp -r com.cocoatech.deckr.metainfo.xml /app/share/metainfo/
+      # - mkdir -p /app/share/icons/hicolor/scalable/apps
+      # - cp -r icon.svg /app/share/icons/hicolor/scalable/apps/com.cocoatech.deckr.svg
+      - mkdir -p /app/share/icons/hicolor/32x32/apps
+      - cp -r app_icon_32.png /app/share/icons/hicolor/32x32/apps/com.cocoatech.deckr.png
+      - mkdir -p /app/share/icons/hicolor/64x64/apps
+      - cp -r app_icon_64.png /app/share/icons/hicolor/64x64/apps/com.cocoatech.deckr.png
+      - mkdir -p /app/share/icons/hicolor/128x128/apps
+      - cp -r app_icon_128.png /app/share/icons/hicolor/128x128/apps/com.cocoatech.deckr.png
+      - mkdir -p /app/share/icons/hicolor/256x256/apps
+      - cp -r app_icon_256.png /app/share/icons/hicolor/256x256/apps/com.cocoatech.deckr.png
+      - mkdir -p /app/share/icons/hicolor/512x512/apps
+      - cp -r app_icon_512.png /app/share/icons/hicolor/512x512/apps/com.cocoatech.deckr.png
+      - mkdir -p /app/share/applications
+      - cp -r com.cocoatech.deckr.desktop /app/share/applications/
+      - mkdir -p /app/share/appdata
+      - cp -r com.cocoatech.deckr.metainfo.xml /app/share/appdata/
+    sources:
+      - type: archive
+        url: https://get.cocoatech.com/dashboard/Deckr-Linux-Portable.tar.gz
```

Comment:
so you unpack a binary, does it also have aarch64?

Reply:
I don't know.  It's a flutter desktop app.  Does flutter run on aarch64?  I never tried or even knew about it.

Reply:
how do I build two arches? Are they separate downloads?  Is there a simple example on how to do this?